lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 69px;
+$nx-sidebar-logo-height: 33px;
+$nx-sidebar-nav-background-color: $nx-blue-1000;
+$nx-sidebar-nav-text-color: $nx-white;
+$nx-sidebar-nav-help-color: $nx-grey-50;
+$nx-sidebar-nav-hover-color: $nx-indigo-600;
+$nx-sidebar-nav-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+
+.nx-sidebar-nav {
+  @include container-vertical;
+
+  box-sizing: border-box;
+  background-color: $nx-sidebar-nav-background-color;
+  color: $nx-sidebar-nav-text-color;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  min-width: 250px;
+  padding: 0;
+  padding-top: $nx-spacing-l;
+  position: relative;
+  // transition: all 0.5s ease;
```

Comment:
Do we want to add any sort of transition for the toggling of the sidebar? If so I could use some help smoothing it out: with the links' text suddenly appearing the transition looks _jumpy_.

Reply:
> If so I could use some help smoothing it out: with the links' text suddenly appearing the transition looks jumpy.

Could have the text slide in from the right as the sidebar expands.  Just an idea, not sure if it would be good.  Also I don't recommend using `all` for the transition property.  I feel like that could very easily effect properties we don't intend it to

Reply:
> Could have the text slide in from the right as the sidebar expands

How can I do that?

> I don't recommend using all for the transition property

Which properties do you think we should transition?

Reply:
> Which properties do you think we should transition?

That depends on the animation which is ultimately up to the designer.  But only the exact ones that the animation actually needs to transition.  So most likely `width` and maybe a small number of others.

> How can I do that?

It would take some fiddling; I wouldn't spend time on it unless its decided upon as the desired effect.

Reply:
> > Which properties do you think we should transition?
> 
> That depends on the animation which is ultimately up to the designer. But only the exact ones that the animation actually needs to transition. So most likely `width` and maybe a small number of others.
> 
> > How can I do that?
> 
> It would take some fiddling; I wouldn't spend time on it unless its decided upon as the desired effect.

Removed transition for now. 55e1df8284ca6704b9e34378dcd7b38f214fd082